django/db/migrations/operations/models.py

```diff
@@ -43,6 +53,11 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
         self.bases = bases or (models.Model,)
         self.managers = managers or []
         super(CreateModel, self).__init__(name)
+        # Sanity-check that there are no duplicated field names, bases, or
+        # manager names
+        check_for_duplicates((name for name, _ in self.fields), "field")
+        check_for_duplicates(self.bases, "base")
```

Comment:
@charettes in many cases `models.Model` is used as the base (it's the default) and that does not have a `_meta` attribute - is this something that can safely be special-cased? or could it be referenced by a dotted-path string (e.g., 'django.db.models.Model')?